client/src/components/StreamField/blocks/BaseSequenceBlock.js

```diff
@@ -119,9 +119,9 @@ export class BaseSequenceChild extends EventEmitter {
           <div class="c-sf-container__block-container">
```

Comment:
`.c-sf-block__actions` (move up/down, duplicate/delete) still use the icon font

this would be `ActionButton.render()` around line 17 in this file.

Reply:
Done with https://github.com/wagtail/wagtail/pull/7565/commits/5b1999eeafa5566b8da8192643516d34228e83f0. By the way, icons are now centered in the button.